docs/tutorials/pytorch2onnx.md

```diff
@@ -23,12 +23,9 @@
 
 ### Prerequisite
 
-1. Please refer to [get_started.md](../get_started.md) for installation of MMCV and MMDetection.
-2. Install onnx and onnxruntime
-
-  ```shell
-  pip install onnx onnxruntime
-  ```
+1. Please install the prerequisites except for MMCV and MMDetection refer to steps 1-3 in [get_started.md/Installation](../get_started.md).
```

Comment:
1.  Install the prerequisites following steps 1-3 in [get_started.md/Installation](../get_started.md) (we do not install MMCV and MMDetection because we will do it manually).